recipes/phykit/meta.yaml

```
@@ -0,0 +1,265 @@
+    - pk_monophyly_check = phykit.phykit:monophyly_check
+    - pk_is_monophyletic = phykit.phykit:monophyly_check
+    - pk_nearest_neighbor_interchange = phykit.phykit:nearest_neighbor_interchange
+    - pk_nni = phykit.phykit:nearest_neighbor_interchange
+    - pk_patristic_distances = phykit.phykit:patristic_distances
+    - pk_pd = phykit.phykit:patristic_distances
+    - pk_polytomy_test = phykit.phykit:polytomy_test
+    - pk_polyt_test = phykit.phykit:polytomy_test
+    - pk_ptt = phykit.phykit:polytomy_test
+    - pk_polyt = phykit.phykit:polytomy_test
+    - pk_print_tree = phykit.phykit:print_tree
+    - pk_print = phykit.phykit:print_tree
+    - pk_pt = phykit.phykit:print_tree
+    - pk_prune_tree = phykit.phykit:prune_tree
+    - pk_prune = phykit.phykit:prune_tree
+    - pk_rename_tree_tips = phykit.phykit:rename_tree_tips
+    - pk_rename_tree = phykit.phykit:rename_tree_tips
+    - pk_rename_tips = phykit.phykit:rename_tree_tips
+    - pk_rf_distance = phykit.phykit:rf_distance
+    - pk_robinson_foulds_distance = phykit.phykit:rf_distance
+    - pk_rf_dist = phykit.phykit:rf_distance
+    - pk_rf = phykit.phykit:rf_distance
+    - pk_root_tree = phykit.phykit:root_tree
+    - pk_root = phykit.phykit:root_tree
+    - pk_rt = phykit.phykit:root_tree
+    - pk_spurious_sequence = phykit.phykit:spurious_sequence
+    - pk_spurious_seq = phykit.phykit:spurious_sequence
+    - pk_ss = phykit.phykit:spurious_sequence
+    - pk_terminal_branch_stats = phykit.phykit:terminal_branch_stats
+    - pk_tbs = phykit.phykit:terminal_branch_stats
+    - pk_tip_labels = phykit.phykit:tip_labels
+    - pk_labels = phykit.phykit:tip_labels
+    - pk_tree_labels = phykit.phykit:tip_labels
+    - pk_tl = phykit.phykit:tip_labels
+    - pk_tip_to_tip_distance = phykit.phykit:tip_to_tip_distance
+    - pk_t2t_dist = phykit.phykit:tip_to_tip_distance
+    - pk_t2t = phykit.phykit:tip_to_tip_distance
+    - pk_tip_to_tip_node_distance = phykit.phykit:tip_to_tip_node_distance
+    - pk_t2t_node_dist = phykit.phykit:tip_to_tip_node_distance
+    - pk_t2t_nd = phykit.phykit:tip_to_tip_node_distance
+    - pk_total_tree_length = phykit.phykit:total_tree_length
+    - pk_tree_len = phykit.phykit:total_tree_length
+    - pk_treeness = phykit.phykit:treeness
+    - pk_tness = phykit.phykit:treeness
+    - pk_saturation = phykit.phykit:saturation
+    - pk_sat = phykit.phykit:saturation
+    - pk_treeness_over_rcv = phykit.phykit:treeness_over_rcv
+    - pk_toverr = phykit.phykit:treeness_over_rcv
+    - pk_tor = phykit.phykit:treeness_over_rcv
+    - pk_create_concatenation_matrix = phykit.phykit:create_concatenation_matrix
+    - pk_create_concat = phykit.phykit:create_concatenation_matrix
+    - pk_cc = phykit.phykit:create_concatenation_matrix
+    - pk_thread_dna = phykit.phykit:thread_dna
+    - pk_pal2nal = phykit.phykit:thread_dna
+    - pk_p2n = phykit.phykit:thread_dna
+  script: "{{ PYTHON }} -m pip install . -vv"
```

The standard incantation is:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```